wasm/jit/jit_amd64_test.go

```diff
@@ -23,137 +23,6 @@ import (
 	"github.com/tetratelabs/wazero/wasm/internal/wazeroir"
 )
 
-// TODO: have some utility functions to reduce loc here: https://github.com/tetratelabs/wazero/issues/100
```

Comment:
these are moved to `jit_test.go` so we can reuse the utilities between arm64 and amd64!

Reply:
other than that no change in this file

Reply:
thanks!